docs/developer-docs/latest/setup-deployment-guides/installation/cli.md

```diff
@@ -63,6 +79,10 @@ If you want to use specific database, you don't have to use the `--quickstart` f
 By default, `create-strapi-app` will generate an empty Strapi project. If you want an application that is pre-configured for a specific use case, see the [Templates section](/developer-docs/latest/setup-deployment-guides/installation/templates.md).
 :::
 
+:::tip
+When using the `create strapi-starter` command, the project can be created from a specific branch, for example with a url like this: `https://github.com/strapi/strapi-starter-gatsby-blog/tree/<my-branch>`.
```

Comment:
`create-strapi-starter` the spacing is `create strapi-starter` is specific to yarn, explained here:

https://classic.yarnpkg.com/en/docs/cli/create/

Reply:
Alright, thanks for your feedback, both of your comments have been fixed with [this commit](https://github.com/strapi/documentation/pull/212/commits/bb8ca3d7461b775faa4e14d661a51e498f239077)